settings/settings.go

```diff
@@ -0,0 +1,36 @@
+package settings
+
+import (
+	"log"
+	"time"
+)
+
+type setterFunc func(string) error
+
+type HubCacheSettings struct {
+	CacheEnabled   *bool
```

Comment:
this is a HubCacheSettings struct - we do not need the word cache in the property names
`Enabled`
`Ttl`
`ClearTime`